TypeCobol/Compiler/Diagnostics/CrossChecker.cs

```diff
@@ -36,6 +36,7 @@ public override bool BeginNode(Node node)
             }
 
             RedefinesChecker.OnNode(node);
+            StructureChecker.OnNode(node);
```

Comment:
Your code is consistent with our other checkers.
However, our new checker should use a more derivated method like `Visit(DataDefinition)`.